src/vs/workbench/contrib/webview/browser/pre/main.js

```diff
@@ -459,9 +459,13 @@ const applyStyles = (document, body) => {
 	}
 
 	if (body) {
-		body.classList.remove('vscode-light', 'vscode-dark', 'vscode-high-contrast', 'vscode-reduce-motion', 'vscode-using-screen-reader');
```

Comment:
not sure what we should do about the below `body.dataset.vscodeThemeKind = initData.activeTheme;` line.